DOCSP-35309 add Kafka to intro installation page (#97)
mongodb/docs-relational-migrator: source/installation.txt

     - Description

   * - Local

     - Suitable for evaluation and proof of concept jobs. 

       For more information, see :ref:`Install on a Single Machine 
       <single-machine-install>`. 

   * - Unattended server

     - Suitable for smaller production jobs. 
       
       For more information, see :ref:`Install on an Unattended Server 
       <unattended-server>`. 

   * - Kafka

     - The Kafka deployment method is recommended for long-running Change Data 

>        Capture (CDC) syncs and large data migrations of more than 20GB. Apache Kafka 

Reviewer: ```suggestion
       Capture (CDC) sync jobs and large data migrations of more than 20GB. Apache Kafka 
```

Author: Updated.

>        Capture (CDC) syncs and large data migrations of more than 20GB. Apache Kafka 

Reviewer: **Suggestion**

Link for additional context

```suggestion
       Capture (CDC) syncs and large data migrations of more than 20GB. `Apache Kafka <https://kafka.apache.org/>`__
```

Author: Updated.

       is an open-source platform for distributed workloads that offers high 
       availability and automatic recovery. This option is the most advanced, and is 
       intended for users who require long-running or large production data syncs. 
       

>        For more information, see :ref:`kafka-intro`.

Author: [For reviewer]

This is causing a build error but will be resolved once https://github.com/mongodb/docs-relational-migrator/pull/96 is merged.

   * - Kafka with Confluent Cloud

     - The Kafka with Confluent Cloud deployment method is an option for users who 
       want the reliability of Kafka without having to manage their own cluster. 

>        Confluent is a fully-managed Kafka hosting provider run by the founders of 

Reviewer: **Suggestion**

Link for additional context:

```suggestion
       `Confluent <https://www.confluent.io/>`__ is a fully-managed Kafka hosting provider run by the founders of 
```

Author: Updated.

       Kafka. This deployment method uses APIs to add the newest Relational Migrator 

>        sink connectors to Confluent Cloud as managed connectors.

Reviewer: **Question**

Should this be `sync`? Also will we have surface area in the docs for explaining the context of a sink/sync connector?

```suggestion
       sink connectors to Confluent Cloud as managed connectors.
```

Author: In this case it is `sink`. Currently we don't have anything major but I could add a sentence to the Kafka landing page where it defines sink connector as "an application that reads data from Kafka"?

Reviewer: That SGTM, just a slight glossary type entry to denote what it is exactly should be fine.

>        For more information, see :ref:`install-confluent-kafka`.

Author: [For reviewer]

This is causing a build error but will be resolved once https://github.com/mongodb/docs-relational-migrator/pull/96 is merged.

     - Description

   * - Local

     - Suitable for evaluation and proof of concept jobs. 

       For more information, see :ref:`Install on a Single Machine 
       <single-machine-install>`. 

   * - Unattended server

     - Suitable for smaller production jobs. 
       
       For more information, see :ref:`Install on an Unattended Server 
       <unattended-server>`. 

   * - Kafka

     - The Kafka deployment method is recommended for long-running Change Data 
       Capture (CDC) sync jobs and large data migrations of more than 20GB. `Apache 
       Kafka <https://kafka.apache.org/>`__ is an open-source platform for distributed 
       workloads that offers high availability and automatic recovery. This is the most 
       advanced option that is intended for users who require long-running or large 

>        production data syncs. 

Reviewer: **Consistency**

```suggestion
       production data migrations. 
```
